server/events/vcs/github_client.go

```diff
@@ -501,3 +501,7 @@ func (g *GithubClient) DownloadRepoConfigFile(pull models.PullRequest) (bool, []
 func (g *GithubClient) SupportsSingleFileDownload(repo models.Repo) bool {
 	return true
 }
+
+func (g *GithubClient) GetCloneURL(VCSHostType models.VCSHostType, repo string) (string, error) {
+	return "", fmt.Errorf("not yet implemented")
```

Comment:
Is this needed for plan/apply endpoints to work for github? If so can we also prioritize this before merging? Most users of Atlantis are probably on GitHub

Reply:
Thanks for the feedback! Implemented.